Automattic/woocommerce-payments
# Add progressive onboarding eligibility modal

Fixes #6140 

#### Changes proposed in this Pull Request
- Add new SVG illustration.
- Add some more text style mixins, renaming the existing one to add a wp/wc prefix.
- Implement the eligibility modal on the overview page on connection success when the account has PO enabled but not completed.
- Use `wcpaySettings.connectUrl` with `collect_payout_requirements` param to ensure Stripe KYC only asks for due information.

#### Testing instructions
>**Note** To ensure the account onboard using PO, you can change server method `can_merchant_use_progressive_onboarding` to always return true.

- Onboard a new PO account.
- After being redirected to **Payments → Overview** you should see the new modal.
- Clicking on **Enable payments only** should dismiss the modal.
- Refresh to get it again – It's expected because we keep `wcpay-connection-success` param.
- Clicking on **Set up payments and deposits** should redirect to Stripe KYC, where only payout information should be asked.
- After being redirected back the modal **should not** be displayed.

>**Warning** Until https://github.com/Automattic/woocommerce-payments-server/pull/3301 is merged, the modal will be still visible after PO completion.

#### Screenshot
<img width="711" alt="Screenshot 2023-05-08 at 11 39 14" src="https://user-images.githubusercontent.com/7670276/236791113-30800fa7-d898-429a-b89e-0694868d0519.png">

-------------------


**Post merge**
